sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -123,9 +133,9 @@ object functions {
    * can handle parameterized scala types e.g.: List, Seq and Map.
    *
    * @group normal_funcs
-   * @since 2.2.0
+   * @since 3.2.0
    */
-  def typedLit[T : TypeTag](literal: T): Column = literal match {
+  def typedlit[T : TypeTag](literal: T): Column = literal match {
```

Comment:
not `typed_lit` but `typedlit`?

Reply:
For some reasons I like `typedlit` (like `datediff`, `shiftleft, etc.). But I don't mind changing it back if other people prefer `typed_lit` more. It's not even in the function registry in any event so I think we can pick one.

Reply:
Ah, I see. I just wanted to know the reason. The current one looks fine.